src/components/app.tsx

```diff
@@ -0,0 +1,13 @@
+import Main from '../pages/main/main';
+
+type AppProps = {
+  dataMain: {
+     RentalOffers: number;
+   };
+}
+
+export default function App({ dataMain }: AppProps): JSX.Element {
+  return (
+    <Main dataMain={dataMain} />
```

Comment:
Название слишком избыточное, проп с данными для компонента `Main` и без указания этого в названии понятен ¯\_(ツ)_/¯ 

```suggestion
    <Main data={data} />
```

Reply:
окей